.ci/build-app-image.sh

```diff
@@ -26,6 +30,8 @@ ENV DNX_TRACE 1
 ENTRYPOINT sleep 10000 | dnx . kestrel
```

Comment:
@kichalla problem is, if you actually checkout to 1.0.0-beta4 tag for instance, they don't have dockerfiles: https://github.com/aspnet/Home/tree/v1.0.0-beta4/samples/1.0.0-beta4 yet we build them in the CI as well... 

So keeping this should be making sense as it injects dockerfiles to old versions where it has to be "`dnx . kestrel`"... 

Maybe it's time for us  to retire all those old versions (beta1-2-3-4) today and get going with the newer ones. This could clarify and make the CI scripts a lot precise.